cockatrice/src/game/player/player.cpp

```diff
@@ -839,7 +839,7 @@ void Player::retranslateUi()
         sayMenu->setTitle(tr("S&ay"));
     }
 
-    aSelectAll->setText(tr("&Select All in Zone"));
```

Comment:
here is some background on what the & does: when opening a menu it automatically adds a shortcut with alt + the following letter, eg if a card menu were opened one would be able to use alt s to select this action (or open a sub menu), if two ampersands precede the same letter then instead of immediately selecting that one it'll cycle through them, requiring another button press to select the one you want, this is why we sometimes have it in front of a random letter, on mac os this feature is disabled (similar to the showing of shortcuts, god forbid mac users figure out how to use the keyboard), we of course don't actually know how many people are aware or use this feature at all but if I remember correctly it makes the use of the menus more accessible to users that have difficulty with pointing devices.

Reply:
Should I just put it in front of a random letter then? I'm not sure what happens if two actions have overlapping auto-hokeys or if they conflict with existing keyboard shortcuts

Reply:
either works, it doesn't hurt to have it overlap, it's just an extra keypress, imo some of the random keys we use make so little sense that they take more time to use, reading the text looking for the highlighted letter, than an extra keypress

Reply:
afaik the menu takes focus when it is open, eating all shortcuts